defaults/main.yml

```diff
@@ -151,6 +151,7 @@ bbb_breakout_rooms_privatechat_enabled: true
 bbb_user_inactivity_inspect_timer: 0
 bbb_user_inactivity_threshold: 30
 bbb_user_activity_sign_response_delay: 5
+bbb_learningdashboardenabled: true
```

Comment:
(I saw camelCase not working.) Maybe use dashes like with the other variables to improve readability?
`bbb_learning_dashboard_enabled`